packages/frontend/src/hooks/useBlacklistedTokens.ts

```diff
@@ -1,35 +1,22 @@
-import {
-    type ChainId,
-    type FeaturedBlacklistedKPITokens,
-} from "@carrot-kpi/sdk";
-import { useQuery } from "@tanstack/react-query";
+import { type ChainId } from "@carrot-kpi/sdk";
 import type { Address } from "viem";
-import { STATIC_CDN_URL } from "../constants";
+import { useFeaturedBlacklistedKPITokenAddresses } from "./useFeaturedKPITokenAddresses";
```

Comment:
Should we rename this file to `useFeaturedBlacklistedKPITokenAddresses`?

Reply:
Whoops, defo. Addressed in a950677158964d96c47cdd65d8cf1690d807720f